phsyn-t3/cod-u8776t_phsyn-t3.xml

```diff
@@ -57,881 +57,1053 @@
-            tantum risibile; ergo tantum homo’. Ergo hec est vera ‘Sortes videt tantum omnem
-            hominem non videntem se’. Item ad idem. Posito quod sint tantum tres
-            homines ceci et omnes alii videant se, tunc iste due convertuntur ‘omnis homo
-            cecus’ et ‘omnis homo non videns se’. Sed possibile est Sortem videre
-            tantum22 omnem hominem cecum. Ergo possibile est Sortem videre tantum 24 omnem
-            hominem non videntem se. Ergo prima possibilis est. </p>
-          <p xml:id="phsyn-t3-d1e378">Solutio. Dico quod prima est falsa et impossibilis, sicut
-            prius dictum est, quia ponit contradictionem, sicut prius patuit duplici ratione. Ad
-            priman autem positionem dicendum quod ponit duo incompossibilia4 .Et concedo quod
-            utrumque eorum est possibile secundum se sed 6 ambo insimul sunt incompossibilia.
-            Quia hoc est possibile secundum se, scilicet quod tantum sint illi tres homines
-            non videntes se, et aliud similiter est possibile secundum se, scilicet quod Sortes
-            videat tantum illos tres. <pb ed="#T" n="33-r"/>
+
+          <p xml:id="phsyn-t3-d1e372"> <pc type="#pilcrow"/>
+          <lb ed="#T"/>Item. Illud idem ostenditur alia ratione, quia si Sortes non videt aliquem
+          <lb ed="#T"/>hominem videntem se, ergo Sortes videt nullum hominem videntem se, quia
+            <lb ed="#T"/> ‘non aliquem’ et ‘nullum’ equipollent, sicut ‘non aliquis’ et ‘nullus’. Ergo nullus homo
+            <lb ed="#T"/>videns se videtur a Sorte (convertendo activam in passivam).
+            <lb ed="#T"/>Ergo Sortes non videt se, quia si Sortes videt se, aliquis homo <add place="right-margin">videns se</add> videtur a Sorte. Et
+            <lb ed="#T"/>dicebatur quod nullus. Ergo a primo: si Sortes non videt aliquem hominem <del rend="strikethrough">non</del>
+            <lb ed="#T"/> videntem se, Sortes non videt se. Et sic habetur alia pars
+            contradictio<lb ed="#T"/>nis. Ex qua cum altera expositiva prime sequitur reliqua pars
+            <lb ed="#T"/> contradictionis, sicut prius. Quare prima habet se contradictionem. Quod concedimus.</p>
+
+          <p xml:id="phsyn-t3-d1e375">
+            <lb ed="#T"/><pc type="#pilcrow"/>Item. Probo quod prima est vera, hec scilicet ‘Sortes videt tantum omnem hominem
+            <lb ed="#T"/>non videntem se’. Ponatur quod Virgilius et Plato et Cicero situ homines
+            <lb ed="#T"/>non videntes se tantum et omnes alii videant se; et ponatur quod Sortes videat tantum illos tres, quia hec positio possibilis est. Et
+            <lb ed="#T"/><seg type="correction"> tunc hec
+                <subst>
+                    <del>cum hoc</del>
+                    <add>tunc hec</add>
+                  </subst>
+                </seg> <!--another hand corrects the lesson by ammending the existing letters into new ones!-->
```

Comment:
again you do not need the `<seg>`

It should be `<subst><del>cum hoc</del><add>tunc hec</add></subst>`

If you want to indicate that the addition is from a different hand then we can add a `@hand` attribute. E.g. `<add hand="#hand1">`